qualtran/bloqs/rotations/phase_gradient.py

```diff
@@ -135,10 +135,8 @@ class AddIntoPhaseGrad(GateWithRegisters, cirq.ArithmeticGate):
         U|x\rangle|\text{phase\_grad}\rangle = |x\rangle|\text{phase\_grad} + x\rangle
     $$
 
-    TODO(#654): Check whether phase_bitsize >= inp_bitsize needs to be enforced.
```

Comment:
Should probably capture the conclusion of this todo in the args somewhere.

Reply:
See the newly added and very detailed `qualtran/bloqs/rotations/quantum_variable_rotation.ipynb` notebook.